### DISCUSSION TITLE
Bambu Lab X1E

### DISCUSSION THREAD

**Discussion by beathoch** [greghesp/ha-bambulab#319] [Ideas]
Hello,
For my purchase decision: Is it planned to integrate the Bambu Lab X1E into the interface?
I am particularly important for the control and monitoring of installation space heating.

**Comment by AdrianGarside** [comment]
Yes, we will if there are no blockers. Given past experiences it should be pretty easy but we need a payload dump from the integration to confirm that everything is as we expect it to be and to understand how any X1E specific payload (such as the space heating) is exposed.

**Reply by beathoch** [reply]
Thats great, thank you for your work!
